tests/testthat/test-fcm.R

```diff
@@ -1,31 +1,42 @@
 context('Testing fcm*.R')
 
-test_that("compare the output feature co-occurrence matrix to that of the text2vec package", {
-    skip_if_not_installed("text2vec")
-    library("text2vec")
-    
+# test_that("compare the output feature co-occurrence matrix to that of the text2vec package", {
```

Comment:
Our diagonals are different but does it make sense to remove this comparison? (They match, except the diagonal.)

If we do remove this code then we can remove **text2vec** from `Suggests:`.